components/script/dom/bindings/codegen/parser/WebIDL.py

```diff
@@ -3686,12 +3687,16 @@ def handleExtendedAttribute(self, attr):
     def getExtendedAttribute(self, name):
         return self._extendedAttrDict.get(name, None)
 
+    def exposedSet(self):
+        return self.exposed
```

Comment:
Note: since parser/WebIDL.py is synced from https://searchfox.org/mozilla-central/source/dom/bindings/parser/WebIDL.py, please copy these changes into a patch in components/script/dom/bindings/codegen/parser and ensure it gets applied in https://github.com/servo/servo/blob/master/components/script/dom/bindings/codegen/parser/update.sh as part of the update script.